# desired pose and velocity of the control joint
desiredPoseInput: [0.0, 1.25, 0.0, 0.0, 0.0, 0.0]
desiredVeloInput: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
# initial index of the DOF associated with the control joint
controlJointDOF0: 0
# cost function weights
## weight on deviations in x and y directions
w1: 1e2
## weight on deviations in z and orientation
w2: 1e0
## weight on virtual stiffness
w3: 5e-3
## weight on final velocities
w4: 1e2
